doc/classes/RenderingDevice.xml

```diff
@@ -2044,28 +2044,34 @@
 		<constant name="DYNAMIC_STATE_STENCIL_REFERENCE" value="64" enum="PipelineDynamicStateFlags" is_bitfield="true">
 		</constant>
 		<constant name="INITIAL_ACTION_CLEAR" value="0" enum="InitialAction">
+			Start rendering and clear the whole framebuffer, region or not (supply appropriate parameters when using this action).
```

Comment:
```suggestion
			Start rendering and clear the whole framebuffer.
```
If ``INITIAL_ACTION_CLEAR`` is used then the whole framebuffer is cleared, the region parameter is ignored